tools/isobuild/meteor-npm.js

```diff
@@ -777,7 +777,7 @@ Profile("meteorNpm.runNpmCommand", function (args, cwd) {
           });
         }
       );
-    });
+    }).await();
```

Comment:
Yeah, I too trust @abernix, which is why I feel really weird about it. But I'm going to let this particular bone go.